lib/hysync/marc_synchronizer/marc_hyacinth_record.rb

```diff
@@ -38,8 +39,8 @@ class MarcHyacinthRecord
       def initialize(marc_record = nil, holdings_marc_records = [], default_digital_object_data = {})
         @errors = []
         @digital_object_data = default_digital_object_data
-        @mapping_ruleset = MarcSelector.first(marc_record, 965, {a: '965hyacinth'})['b']
-
+        hyacinth_flag = MarcSelector.first(marc_record, 965, {a: '965hyacinth'})
+        @mapping_ruleset = hyacinth_flag['b'] if hyacinth_flag
```

Comment:
I forgot -- is it a problem that `@mapping_ruleset` will potentially be an undefined variable?  I don't remember whether passing an undefined variable to a method will result in an error, or will just be nil.

Reply:
variables would be a problem, but attributes are nil

Reply:
this is to accommodate durst, btw

Reply:
ruby also has some slightly weird rules about variable definition we should talk about sometime that are relevant to conditionals.

Reply:
Okay cool.  That's good then.  And talking over variable definition stuff sound good!